src/navigation/Nav.jsx

```diff
@@ -29,20 +30,37 @@ export const DropdownLoader = ({ label }) => (
  * @returns {React.element} Navigation Bar
  */
 export class Nav extends React.Component {
+	/**
+	 * @constructor
+	 * @param {Object} props properties passed into component
+	 */
+	constructor(props) {
+		super(props);
+		this.onDismissSignUpModal = this.onDismissSignUpModal.bind(this);
+		this.onClickSignUpAction = this.onClickSignUpAction.bind(this);
+		this.state = { isSignUpModalOpen: false };
```

Comment:
nitpick - i'm seeing `SignUpModal` used in a lot of these functions or props, but we call the component itself `SignupModal`. maybe worth it to choose one and stick with it